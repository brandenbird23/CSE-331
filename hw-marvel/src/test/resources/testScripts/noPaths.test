# Tests for finding a path between two nodes when there arent any paths

LoadGraph nfl nflPlayers.csv

FindPath nfl GENO-SMITH BROCK-PURDY