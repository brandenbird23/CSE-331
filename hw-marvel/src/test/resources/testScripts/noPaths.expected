# Tests for finding a path between two nodes when there arent any paths

loaded graph nfl

path from GENO-SMITH to BROCK-PURDY:
no path found